CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus v9.5.6 (July 8, 2021) ###
+
+* Makes `citus_prepare_pg_upgrade` idempotent by dropping a transition table
```

Comment:
```suggestion
* Fixes minor bug in `citus_prepare_pg_upgrade` that caused it to loose its idempotency
```

Reply:
Each line can have at most 80 characters. This one has 87.

How does the following sound? It has 78 chars.

```suggestion
* Fixes `citus_prepare_pg_upgrade` bug that caused it to lose its idempotency
```

Reply:
sure thing, bit sad that we lose clearity on the severity of the bug due to an imposed character limit, but if that is the case let it be.

Reply:
Why don't we use multiple lines?

Reply:
I checked with @gurkanindibay and confirmed that multi-line changelog entries work on our packaging infrastructure now.

Updated the entry with @thanodnl 's suggestion split into two lines.